go_deps.bzl

```diff
@@ -38,6 +38,14 @@ def swift_bazel_go_dependencies():
         sum = "h1:fdDeAqgT47acgwd9bd9HxJRDmc9UAmPpc+2m0CXv75Q=",
         version = "v4.7.1",
     )
+    maybe(
```

Comment:
go_deps.bzl shouldn't be necessary with MODULE.bazel right? I thought this was only required for WORKSPACE setups?

Reply:
You are correct. I removed it and the target that generates it. Thanks!